hs_core/models.py

```diff
@@ -1975,19 +1975,39 @@ def get_xml(self, pretty_print=True):
         return self.XML_HEADER + '\n' + etree.tostring(RDF_ROOT, pretty_print=pretty_print)
 
     def add_metadata_element_to_xml(self, root, md_element, md_fields):
+        """
+        helper function to generate xml elements for a given metadata element that belongs to
+        'hsterms' namespace
+
+        :param root: the xml document root element to which xml elements for the specified metadata element needs
+                     to be added
+        :param md_element: the metadata element object
+        :param md_fields: attribute names of the metadata element (if the name to be used in generating the xml element
+                          name is same as the attribute name then it must be a list of attribute names. if xml element
+                          name needs to be different from the attribute name then it must be a list of tuples with
+                          first element in each tuple being the attribute name and the second element being
+                          what will be used in naming the xml element)
+
+        """
         from lxml import etree
 
+        # if the list of attributes is not a list of tuples make it a list of tuples
+        if not isinstance(md_fields[0], tuple):
+            md_fields = ([(k, k) for k in md_fields])
```

Comment:
Don't force _all_ of them to be tuples if the first is. Allow a mix so you can do things like ["field1", ("field2", "rename"), "field3"]